chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -27,6 +27,46 @@
 upgrade_step = node["crowbar_wall"]["crowbar_upgrade_step"] || "none"
 
 case upgrade_step
+
+when "crowbar_upgrade"
+
+  # Disable openstack services
+  # We don't know which openstack services are enabled on the node and
+  # collecting that information via the attributes provided by chef is
+  # rather complicated. So instead we fall back to a simple bash hack
+
+  bash "disable_openstack_services" do
+    code <<-EOF
+      for i in /etc/init.d/openstack-* \
+               /etc/init.d/openvswitch-switch \
+               /etc/init.d/ovs-usurp-config-* \
+               /etc/init.d/drbd;
+      do
+        if test -e $i
+        then
+          initscript=`basename $i`
+          chkconfig -d $initscript
+        fi
+      done
+
+      if systemctl status pacemaker.service 2>&1 > /dev/null
+      then
+        systemctl disable pacemaker.service
```

Comment:
1. `systemctl status` checks if service is running or enabled?

2. I think that we should replace all services in `disable_openstack_services` block with systemd ones, isn't SLES12 systemd-based system?